app/core/templates/core/submission_form.html

```diff
@@ -1,21 +1,19 @@
 {% extends 'layout.html' %}
 {% load crispy_forms_tags %}
 {% block content %}
+<h1>Submission</h1>
 <div class="errors">
   {{ container_form|as_crispy_errors }}
   {{ submission_form|as_crispy_errors }}
+  {{ arg_formset|as_crispy_errors }}
 </div>
-<form method="post" class="form" action="{{ form_action }}">
+<form method="post" class="form" action="{{ form_action }}" enctype="multipart/form-data">
   {% csrf_token %}
-  <div class="border container mt-2">
-    <h2>Container</h2>
-    {{ container_form|crispy }}
+  <div id="accordion">
+    {% include "core/_collapsible_form.html" with label="container" form=container_form show=show_container only %}
+    {% include "core/_collapsible_form.html" with label="args" title="Special Arguments" formset=arg_formset helper=arg_helper show=show_args card_contents="core/_arg_helptext.html" only %}
```

Comment:
This is the template that uses the show_args context variable to set 'show' on the collapsible form template